### DISCUSSION TITLE
It's not possible to tell which target framework a test summary is for

### DISCUSSION THREAD

**Discussion by adamralph** [Tyrrrz/GitHubActionsTestLogger#13] [General]
E.g. https://github.com/adamralph/bullseye/actions/runs/2298612469

The tests are running for `<TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>`, but all the test summaries look the same:

![image](https://user-images.githubusercontent.com/677704/167559284-e41cf73c-2264-4388-8808-5caf0f08918d.png)

I'm not sure exactly where the problem lies, but I thought I'd start by asking here.

**Comment by Tyrrrz** [comment]
Yup, I noticed that too. I'm working on a new version of the summary which will be delivered in a new major. It includes multiple reworks apart from this too. I'm really busy lately so can't promise you an exact estimate, but it's going to come :)
